src/main/resources/META-INF/plugin.xml

```diff
@@ -141,9 +141,10 @@
     <implicit.resource.closer
       implementation="de.plushnikov.intellij.plugin.extension.LombokCleanUpImplicitResourceCloser"/>
 
-    <!--    <referencesSearch implementation="de.plushnikov.intellij.plugin.extension.LombokReferenceSearcher"/>-->
-    <findUsagesHandlerFactory
-      implementation="de.plushnikov.intellij.plugin.extension.LombokFieldFindUsagesHandlerFactory"/>
+    <referencesSearch implementation="de.plushnikov.intellij.plugin.extension.LombokReferenceSearcher"/>
+    <methodReferencesSearch implementation="de.plushnikov.intellij.plugin.extension.LombokMethodReferenceSearcher"/>
+<!--    <findUsagesHandlerFactory-->
```

Comment:
I think findUsagesHandlerFactory will be completely unnecessary if you get referenceSearcher to work.

Reply:
I found that if I add a call to `MethodReferencesSearch.search()` for each lombok-generated getter/setter methods in `LombokReferenceSearcher`, then the resulting behavior is the same as enabling both `LombokReferenceSearcher` and `LombokReferenceSearcher` together.